packages/source-fetcher/lib/sourcify.ts

```diff
@@ -59,6 +59,8 @@ const SourcifyFetcher: FetcherConstructor = class SourcifyFetcher
     "avalanche",
     "fuji-avalanche",
     "wagmi-avalanche",
+    //sourcify does *not* support non-testnet dfk?
```

Comment:
I'm confused by this question in `sourcify.ts` that precedes something named, `testnet-dfk-avalanche`.

Reply:
Yeah so DFK and WAGMI are Avalanche "subnets"?  I don't know what that actually means.  But you'll notice (per my practice of always including the mainnet -- or in this case the mainsubnet? -- in `networks.ts` even if it's not supported yet) I put both `dfk-avalanche` and `testnet-dfk-avalanche` in `networks.ts`.  These are different networks with different chain IDs.  However Sourcify only supports the latter atm?  So yeah I dunno.

Reply:
Thanks @haltman-at